modules/local/spaceranger_count.nf

```diff
@@ -28,8 +27,8 @@ process SPACERANGER_COUNT {
     path "versions.yml"                                      , emit: versions
 
     script:
-    def probeset         = probeset.name != 'EMPTY_PROBESET' ? "--probe-set=${probeset}" : ''
-    def manual_alignment = manual_alignment.name != 'EMPTY_ALIGNMENT' ? "--loupe-alignment=${manual_alignment}" : ''
+    def probeset         = probeset.name != [] ? "--probe-set=${probeset}" : ''
```

Comment:
That's actually the beauty of using `[]`. Groovy evaluates this as false, so you can just do

```
probeset ? "--probe-set=${probeset}" : ""
```

But this part is anyway going to be replaced with the spaceranger from `modules`...

Reply:
Oh, that's quite a bit less verbose! Yeah it'll go away soon, but I'll change it in the meantime anyway.